SlugifierInterface.php

```diff
@@ -9,21 +9,31 @@
  * file that was distributed with this source code.
  */
 
+namespace Symfony\Cmf\Bundle\CoreBundle\Slugifier;
+
+if (!interface_exists('Symfony\Cmf\Bundle\CoreBundle\Slugifier\SlugifierInterface')) {
+    /**
+     * @deprecated Since 1.0, to be removed in 2.0.
+     */
+    interface SlugifierInterface {
```

Comment:
i actually thought of making CoreBundle depend on this library and have this repo contain the paths Bundle/CoreBundle/Slugifier and Api/Slugifier and configure autoloading according. we then also would delete the interface in the CoreBundle repository as its provided here.

that would look less hacky i think.